docker/docker-entrypoint.sh

```diff
@@ -8,8 +8,13 @@ set -e
 DB_WAIT_TIMEOUT=${DB_WAIT_TIMEOUT-3}
 MAX_DB_WAIT_TIME=${MAX_DB_WAIT_TIME-30}
 CUR_DB_WAIT_TIME=0
-NAUTOBOT_DOCKER_SKIP_INIT=${NAUTOBOT_DOCKER_SKIP_INIT-false}
-if [ "$NAUTOBOT_DOCKER_SKIP_INIT" == "false" ]; then
+
+# lowercase NAUTOBOT_SKIP_INIT and assign to SKIP_INIT:
+SKIP_INIT=$(echo $NAUTOBOT_DOCKER_SKIP_INIT | tr '[:upper:]' '[:lower:]')
+# reassign SKIP_INIT to first character of SKIP_INIT:
+SKIP_INIT=${SKIP_INIT:0:1}
+# tests for falsey value ($NAUTOBOT_DOCKER_SKIP_INIT can be any string starting with case-insensitive "f" or "0"):
+if [ "$SKIP_INIT" == "f" ] || [ "$SKIP_INIT" == "0" ]; then
```

Comment:
```suggestion
if [[ "$SKIP_INIT" == "f" ]] || [[ "$SKIP_INIT" == "0" ]]; then
```

I think this is what's needed to do the string comparisons evaluation, otherwise like before it's always evaluating to "true"